apps/services/user-notification/src/app/modules/notifications/notificationsWorker.service.ts

```diff
@@ -30,10 +30,10 @@ export class NotificationsWorkerService implements OnApplicationBootstrap {
   }
 
   async run() {
-    await this.worker.run<Message>(
+    await this.worker.run<CreateHnippNotificationDto>(
       async (message, job): Promise<void> => {
         const messageId = job.id
-        this.logger.info('Message received by worker', { messageId })
+        this.logger.info('Message received by worker ... ...', { messageId })
```

Comment:
What is this addition for?

Reply:
just a little trick to make sure see which version of worker is receiving the message
the ones with ... ... are the new ones